src/forms/formHelpers.js

```diff
@@ -142,9 +141,7 @@ const FormPartsContainer = function(props) {
 * 
 * @returns Component
 */
-const MassiveToggle = function ( props ) {
-
-  var time = props.time || '';
+const MassiveToggle = function (props) {
```

Comment:
Is there a specific reason not to destructure this one?

Reply:
It uses `props.onChange || props.storeChecked`, which means that certain objects would fail destructuring if they passed the wrong parameter. This can be destructured after unifying the names.

Reply:
Sounds good.